lib/locale.ts

```diff
@@ -22,7 +24,7 @@ export function getCanonicalLocale(): string {
  * Returns the user's language
  */
 export function getLanguage(): string {
-	return document.documentElement.lang || 'en'
+	return document.documentElement.lang || environmentLocale
```

Comment:
Language is not locale, probably better:
```suggestion
	return document.documentElement.lang || navigator.language || navigator.userLanguage
```

Reply:
Fixed